doc/index.md

```diff
@@ -71,6 +71,8 @@ Classify points as discrete labels (`0`, `1`, `2`, ...).
  * [`AdaBoost`](user/methods/adaboost.md): Adaptive Boosting
  * [`DecisionTree`](user/methods/decision_tree.md): ID3-style decision tree
    classifier
+ * [`GradBoosting`](user/methods/gradient_boosting.md): gradient boosting
+   classifier
```

Comment:
```suggestion
   classifier built on decision tree regressors
```
Up to you, maybe this clarifies a little bit more about the technique?  If you think it's better as-is feel free to reject the suggestion. :+1:

Reply:
Sure, I'll commit it for now, for clarification. We can remove it later when we get to building a weak learner template.